docs/modules/record-encryption/hashicorp-vault/con-vault-setup.adoc

```diff
@@ -6,7 +6,9 @@
 = Enable the Transit Engine
 
 The filter integrates with the {hashicorp-vault}/docs/secrets/transit[HashiCorp Vault *Transit
-Engine*].   Vault does not enable the Transit Engine by default.  It must be
+Engine*].
+Vault does not enable the Transit Engine by default.
+It must be
 {hashicorp-vault}/docs/secrets/transit#setup[enabled] before it can be used with the filter.
```

Comment:
nit:
```suggestion
{hashicorp-vault}/docs/secrets/transit#setup[enabled] before it can be used by the filter.
```

but its the filter that uses the transit engine, not the transit engine calling back to the filter.